tests/std/tests/P0645R10_text_formatting_formatting/test.cpp

```diff
@@ -47,10 +47,11 @@ struct choose_literal<wchar_t> {
 
 template <class charT, class... Args>
 auto make_testing_format_args(Args&&... vals) {
+    using context = basic_format_context<back_insert_iterator<basic_string<charT>>, charT>;
     if constexpr (is_same_v<charT, wchar_t>) {
-        return make_wformat_args(forward<Args>(vals)...);
+        return make_wformat_args<context>(forward<Args>(vals)...);

```

Comment:
It (incorrectly) does: https://github.com/microsoft/STL/blob/a10865713d9958f48819fea855bb58a357a620eb/stl/inc/format#L2570. I plan to fix `make_wformat_args` in a separate PR since it's unrelated to this change.

Reply:
I've already opened #1853.